symbols/shop/funeral_directors.svg

```diff
@@ -0,0 +1,534 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:i="http://ns.adobe.com/AdobeIllustrator/10.0/"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   i:rulerOrigin="0 0"
+   enable-background="new 0 0 153.46 357.225"
+   xml:space="preserve"
+   i:viewOrigin="229.77 574.1123"
+   i:pageBounds="0 792 612 0"
+   version="1.1"
+   y="0px"
+   x="0px"
+   viewBox="0 0 14 13.999999"
+   id="svg2"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="shop_funeraldirector.svg"
+   width="14.000001"
+   height="14"><defs
+     id="defs47" /><sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="1680"
+     inkscape:window-height="990"
+     id="namedview45"
+     showgrid="false"
+     fit-margin-top="0"
+     fit-margin-left="0"
+     fit-margin-right="0"
+     fit-margin-bottom="0"
+     inkscape:zoom="1.5"
+     inkscape:cx="-162.52074"
+     inkscape:cy="8.711463"
+     inkscape:window-x="0"
+     inkscape:window-y="0"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg2" /><i:pgf
+     id="adobe_illustrator_pgf">
+	eJzsvYlWG0m2Bfp+QP8gMJhZynnABswMNpONwWBjYwGywYCEJVHVdd9a79vf2edESpmpzFRqqNvd
```

Comment:
This uninteresting blob of meta data should be removed. 
The easierst way would be to open the file with inkscape and save as "optimized" svg which removes such things.